### ISSUE TITLE
5.0.0 Cannot install on PCL

### ISSUE THREAD

**Issue by nbelley** [AutoMapper/AutoMapper#1449]
Since version 4.1, it's been impossible for me to update Automapper on my PCL projects.

I tried again with 5.0.0 with an updated Visual Studio and Xamarin to the latest version (Update 3).

Still not working. I'm wondering if I'm doind something wrong or it's simply normal and will never be supported? 

Actual error message: Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not install package 'AutoMapper 5.0.0'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile49', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.			0	

Thanks.

![1](https://cloud.githubusercontent.com/assets/4919080/16596483/299253a4-42c2-11e6-86be-f61630877975.jpg)
![2](https://cloud.githubusercontent.com/assets/4919080/16596482/29918244-42c2-11e6-9b63-df5aab836bdd.jpg)
![3](https://cloud.githubusercontent.com/assets/4919080/16596481/2991710a-42c2-11e6-9849-054b144adfa5.jpg)
![4](https://cloud.githubusercontent.com/assets/4919080/16596484/2994155e-42c2-11e6-8b4e-6b90b5ab397e.jpg)
![5](https://cloud.githubusercontent.com/assets/4919080/16596556/7fe20ae2-42c2-11e6-89d6-04f818d59fe4.jpg)
![6](https://cloud.githubusercontent.com/assets/4919080/16596586/9c5dfbf4-42c2-11e6-954e-adc20427adea.jpg)

**Comment by jbogard** [comment]
Can you try the MyGet feed? I believe this one's good.

**Comment by nbelley** [comment]
> Can you try the MyGet feed? I believe this one's good.

I'm sorry but I have no idea what is the MyGet feed address or configuration, can you help me there?

**Comment by nbelley** [comment]
I tried with 5.0.1-alpha-01144:

> Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not install package 'AutoMapper 5.0.1-alpha-01144'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile49', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.			0

**Comment by jbogard** [comment]
Oh, yeah, I don't support that profile. Profile49 is .NET 4.5 and Windows Phone Silverlight 8. I'm not supporting Silverlight anymore. Are you deploying to Windows Phone Silverlight 8?

**Comment by nbelley** [comment]
Yes, I was trying not to target Silverlight, but if I don't force targeting Windows specific version Silverlight is automatically targeted.

Works! 
Thanks.

**Comment by jbogard** [comment]
So....what platforms do you WANT to target?

**Comment by nbelley** [comment]
Specifically android and ios. But if I target only those, Silverlight gets automatically targeted. To make sure it's not, I have to force a Windows 8 target.

**Comment by jbogard** [comment]
So AutoMapper only supports netstandard1.1 and above. To get to target, you need one of the profiles listed here: https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#portable-profiles

**Comment by nbelley** [comment]
Thanks, the installation worked. 

Can you tell me the target release for the stable version of this build?

**Comment by jbogard** [comment]
This week.

**Comment by jbogard** [comment]
5.0.1 is released!

**Comment by shanempope** [comment]
Putting this here because it's the only issue I could find related when I searched.

I ran into a similar issue. The error I got was "Could not install package 'System.Runtime.InteropServices.RuntimeInformation 4.0.0'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile7', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author."

It was an error in net core https://github.com/dotnet/corefx/issues/10445 

I'm not sure how long it will take for that fix to propagate. So I had to install 
System.Runtime.InteropServices.RuntimeInformation nuget package first, and then AutoMapper 5 would install from nuget just fine. Hope this helps someone else.
